cooker/cooker.py

```diff
@@ -444,10 +446,24 @@ def update_directory(self, method, local_dir, has_remote, branch, rev):
     def generate(self):
         info('Generating dirs for all build-configurations')
 
+        self.read_local_conf_version()
+
         for build in BuildConfiguration.ALL.values():
             if build.buildable():
                 self.prepare_build_directory(build)
 
+    def read_local_conf_version(self):
+        try:
+            self.distro.local_conf_version = "{}".format(self.distro.DEFAULT_CONF_VERSION)
+            file = open(self.config.layer_dir() + self.distro.BASE_DIRECTORY + "/" + self.distro.TEMPLATE_CONF + "/local.conf.sample")
+            for line in file.readlines():
+                if line.lstrip().startswith("CONF_VERSION"):
+                    idx = line.rfind("=")
+                    self.distro.local_conf_version = int(line[idx+1:].replace('"', ''))
+                    return
```

Comment:
here local-version is an int? By default from above it was a string?

```suggestion
            for line in file:
                 if line.lstrip().startswith("CONF_VERSION"):
                    version = re.search(r'\d+', t).group(0)
                    return
```